Makefile.toml

```diff
@@ -0,0 +1,107 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = "true"
+CARGO_MAKE_WORKSPACE_EMULATION = true
+CARGO_MAKE_CRATE_WORKSPACE_MEMBERS = [
+	"crt0stack",
+	"enarx-keep-sgx",
+	"enarx-keep-sgx-shim",
+	"enumerate",
+	"intel-types",
+	"iocuddle",
+	"iocuddle-sgx",
+	"memory",
+	"nolibc",
+	"payload",
+	"sev",
+	"sev-show",
+	"sev-types",
+	"sgx-crypto",
+	"sgx-show",
+	"sgx-types",
+	"span",
+	"testing",
+	"units",
+	"vdso",
+	"vmsyscall"
+]
+
+[tasks.audit]
+install_crate = "cargo-audit"
+command = "cargo"
+args = ["audit"]
```

Comment:
Can't we use this?

https://github.com/sagiegurari/cargo-make/blob/master/src/lib/descriptor/makefiles/rust.toml#L544

Reply:
I didn't see that one! I will remove this.